src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/authorization/UserMapper.kt

```diff
@@ -1,16 +1,13 @@
-package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.controller.mappers
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.authorization
```

Comment:
🇬🇧 / 🇺🇸 ? :trollface:

Reply:
yeh this is my bad. however i've not trained myself to really get out of this habit since i find most libs use american spellings anyway. e.g. https://docs.spring.io/spring-security/site/docs/5.2.x/reference/html/authorization.html

Reply:
😁 totally! I was just trolling